pages/tutorials/[slug].js

```diff
@@ -30,10 +34,50 @@ export async function getStaticProps(context) {
 }
 
 export default function Tutorial({ name, steps, id, slug }) {
+  const [currentStep, setCurrentStep] = useState(0);
+
+  function addStep() {
+    setCurrentStep((prevCurrentStep) =>
+      prevCurrentStep < steps.length ? prevCurrentStep + 1 : prevCurrentStep
+    );
+  }
+
+  function subtractStep() {
+    setCurrentStep((prevCurrentStep) =>
+      prevCurrentStep > 0 ? prevCurrentStep - 1 : 0
+    );
+  }
+
   return (
     <>
       <Headline>{name}</Headline>
-      <TutorialStartCard />
+
+      {currentStep === 0 ? (
+        <TutorialStartCard />
+      ) : currentStep < steps.length ? (
+        <TutorialCard step={steps[currentStep]} />
+      ) : (
+        <Headline>done</Headline>
+      )}
+
+      <ButtonContainer>
+        {currentStep === 0 ? (
+          <>
+            <Link href="/tutorials">
+              <Button isPrimary={false}>back to library</Button>
+            </Link>
+          </>
+        ) : (
+          <Button isPrimary={false} onClick={() => subtractStep()}>
+            prev
```

Comment:
Very optional: why would you have a prev button at step 1? id suggest to either call it just 'back', as previous imo suggests a previous step or rename it at step one

Reply:
Great idea! I now change the button's name conditionally for step 1. Thanks, Mark! 😃